buildstockbatch/base.py

```diff
@@ -227,6 +244,8 @@ def get_clean_column_name(x):
                 return x.lower()
 
             tsdf.rename(columns=get_clean_column_name, inplace=True)
+            if cfg.get("postprocessing", {}).get("timestamps_use_period_beginning", False):
+                BuildStockBatchBase.make_period_beginning(tsdf)
```

Comment:
Might be worth turning this into a [classmethod](https://docs.python.org/3/library/functions.html#classmethod) then you can implicitly reference the class.